### ISSUE TITLE
Android 11 Velociraptor dissapear after some amount of time in Android Auto

### ISSUE THREAD

**Issue by goofy1988** [plusCubed/velociraptor#115]
Hello, sadly in Android 11 Velociraptor starts to disappear after a certain amou t of time while using google maps with android Auto on phone. I checked every settings. Everything is set to be allowed. Battery feartes are diabled for Velociraptor. Anyone else having such issues?

**Comment by FDisk80** [comment]
Yes, it does. Some bug with Android 11 maybe. The app really needs an update.
